### PR TITLE
Upgrade the video.js player to version 5.2.1

### PR DESCRIPTION
**Pull request by mouhb**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
     <name>Video.js</name>
     <description>WebJar for Video.js</description>
     <url>http://webjars.org</url>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <upstream.version>4.11.4</upstream.version>
-        <upstream.url>http://www.videojs.com/downloads</upstream.url>
+        <upstream.version>5.2.1</upstream.version>
+        <upstream.url>https://github.com/videojs/video.js/archive/v5.2.1/video-js-5.2.1.zip</upstream.url>
```
Comment (jamesward) [inline pom.xml]: Use: `https://github.com/videojs/video.js/releases/download/v${upstream.version}/video-js-${upstream.version}.zip`

File: pom.xml [hunk 2]
```diff
@@ -11,15 +11,15 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>video-js</artifactId>
-    <version>4.11.5-SNAPSHOT</version>
+    <version>5.2.1</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `5.2.1-SNAPSHOT` since Maven does the actual release.

File: pom.xml [hunk 3]
```diff
@@ -83,11 +88,13 @@
                         <goals><goal>run</goal></goals>
                         <configuration>
                             <target>
+                                <echo message="download archive" />
+                                <get src="${upstream.url}" dest="${project.build.directory}/${project.artifactId}.zip" />
```
Comment (jamesward) [inline pom.xml]: It is fine to do the download this way but you should remove the `wagon-maven-plugin`.

File: pom.xml [hunk 4]
```diff
@@ -83,11 +88,13 @@
...
                                 <echo message="unzip archive" />
                                 <unzip src="${project.build.directory}/${project.artifactId}.zip" dest="${project.build.directory}" />
                                 <echo message="moving resources" />
                                 <move todir="${destDir}">
-                                    <fileset dir="${project.build.directory}/video-js" />
+                                    <fileset dir="${project.build.directory}/video.js-${upstream.version}" includes="dist/video*.js dist/video*.css dist/video*.swf dist/font/ dist/ie8/ dist/lang/"/>
```
Comment (jamesward) [inline pom.xml]: This might change with the release download.

File: pom.xml (hunk 1 above)
Comment (mouhb) [inline pom.xml]: When I modified the upstream url it generates an issue, As you can see on the link bellow.
https://ncry.pt/p/2lBn#9UlAvmNRHKt6zAAzc7OnNjileuJwr5cAPn-Ydjg6oHI

But if I change it with this one it is working
https://github.com/videojs/video.js/archive/v5.2.1/video-js-5.2.1.zip

**Pull request merged**